File: 151002/Rusakovich/NewsDistributedSystem/src/main/java/by/rusakovich/newsdistributedsystem/model/dto/author/AuthorRequestTO.java
package by.rusakovich.newsdistributedsystem.model.dto.author;

import jakarta.validation.constraints.Size;

public record AuthorRequestTO(
        Long id,
        @Size(min = 2, max = 64) String login,
        @Size(min = 8, max = 128) String password,
        @Size(min = 2, max = 64) String firstname,
        @Size(min = 2, max = 64) String lastname
) {
}

File: 151004/Ermolovich/DC_Lab3/discution/DC_Lab3/DC_Lab1/DTO/AuthorRequestTo.cs
using DC_Lab1.Models;
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class AuthorRequestTo(int Id,string? Login, string? Password, string? Firstname, string? Lastname) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public string? Login { get; set; } = Login;

        public string? Password { get; set; } = Password;

        public string? Firstname { get; set; } = Firstname;

        public string? Lastname { get; set; } = Lastname;

      
    }
}
